host/xtest/regression_6000.c

```diff
@@ -1406,43 +1406,68 @@ static void xtest_tee_test_6012_single(ADBG_Case_t *c, uint32_t storage_id)
 	uint32_t orig;
 	uint32_t obj;
 
+	/*
+	 * create the object a first time (forced through with overwrite attribute)
+	 */
 	if (!ADBG_EXPECT_TEEC_SUCCESS(c,
 		xtest_teec_open_session(&sess, &storage_ta_uuid, NULL, &orig)))
 		return;
 
 	if (!ADBG_EXPECT_TEEC_SUCCESS(c,
 		fs_create_overwrite(&sess, file_04, sizeof(file_04), storage_id)))
-		goto exit;
+		goto bail1;
 
 	TEEC_CloseSession(&sess);
 
-	/* re-create the same */
+	/*
+	 * re-create the object two times with overwrite attribute
```

Comment:
You mean a second time?

Reply:
The sequence below re-creates twice the already existing object. I left it as it was.
Maybe I could say `re-create twice again the object with overwrite attribute`?

Reply:
No it's OK